numbat/src/value.rs

```diff
@@ -23,6 +23,7 @@ impl std::fmt::Display for FunctionReference {
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum Value {
     Quantity(Quantity),
+    Pair(Quantity, Quantity),
```

Comment:
We can go with another approach if there's a cleaner way to do this.

Reply:
We could just return a string for now?

Reply:
Okay, it does that now.